client/state/sites/vouchers/reducer.js

```diff
@@ -28,6 +33,28 @@ import { itemsSchema } from './schema';
 export const items = ( state = {}, action ) => {
 	const { siteId } = action;
 	switch ( action.type ) {
+		case SITE_VOUCHERS_ASSIGN:
+			switch ( action.serviceType ) {
+				case GOOGLE_AD_CREDITS:
+					const currentVouchers = state[ siteId ]
+						? ( state[ siteId ][ GOOGLE_AD_CREDITS ] || [] )
+						: [];
+
+					const vouchers = currentVouchers.splice( 0 );
```

Comment:
is there a reason to `splice( 0 )` here? I'm not sure that it does anything since splice called without elements to add should remove items, but as called here returns the original list, _except that it removes **all** elements of the original list_.

[example](http://jsbin.com/biwasexiyu/edit?js,console)